otherlibs/stdune/src/copyfile_stubs.c

```diff
@@ -0,0 +1,42 @@
+#include <caml/memory.h>
+#include <caml/mlvalues.h>
+
+#if defined(__APPLE__)
+#include <caml/threads.h>
+#include <caml/unixsupport.h>
+#include <copyfile.h>
+
+CAMLprim value stdune_copyfile(value v_from, value v_to) {
+  CAMLparam2(v_from, v_to);
+  copyfile_state_t state = copyfile_state_alloc();
```

Comment:
According to the man page, you can pass `NULL` for the `state` object.

Reply:
Done. Thanks.